src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -764,16 +770,16 @@ bool DirectoryService::RunConsensusOnDSBlockWhenDSPrimary() {
         LOG_GENERAL(INFO,
                     "Node " << pubKeyPoW.first
                             << " failed to join because priority not enough.");
-        if (m_allDSPoWs.find(pubKeyPoW.first) != m_allDSPoWs.end()) {
-          m_allDSPoWs.erase(pubKeyPoW.first);
+        if (allDSPoWs.find(pubKeyPoW.first) != allDSPoWs.end()) {
+          allDSPoWs.erase(pubKeyPoW.first);
```

Comment:
Do we need to reassign allDSPoWs back to m_allDSPoWs?

Reply:
no need. The variable was not used after DSBlockPre. It can be re-used if there is a view change.